modules/serverless-nodejs-function-return-values.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-nodejs-functions.adoc
+
+[id="serverless-nodejs-function-return-values_{context}"]
+= Node.js function return values
+
+Functions can return any valid JavaScript type or can have no return value. When a function has no return value specified, and no failure is indicated, the caller will receive a `204 No Content` response.
+
+Functions may also return a CloudEvent, or a `Message` object in order to push events into the Knative Eventing system. In this case, the developer is not required to understand or implement the CloudEvent messaging specification. Headers and other relevant information from the returned values are extracted and sent with the response.
```

Comment:
s/may/can/
Rewording considerations for clarity if a CloudEvent is a `Message` object:

> Functions can also return a CloudEvent, or a `Message` object, in order to push events into the Knative Eventing system.

If a CloudEvent and a `Message` object are two different things:

> Functions can also return a CloudEvent or a `Message` object in order to push events into the Knative Eventing system.